File: g-input/python/input.py

def take_user_input():
    user_firstname = input("enter your firstname : ")
    user_age = int(input("enter your age : "))

    if(user_firstname == "sofia" and user_age >= 18):
        print(f"Hello {user_firstname}!")
        print("Access to attend a party #tonight")
    else:
        print("sorry not have access")

    return

def main():
    take_user_input()
    return

main()

'''
    output
    ---------
    enter your firstname : Prakash
    enter your age : 27
    sorry not have access

    Hello sofia!
    Access to attend a party #tonight


'''

File: g-input/c++/input.cpp
#include <iostream>
using namespace std;

int main(){
    cout << "enter your firstname : ";
    string user_firstname = "";
    cin >> user_firstname;
    cout << "enter your age : ";
    int user_age;
    cin >> user_age;

    cout << "firstname : " << user_firstname << endl;
    cout << "age : " << user_age << endl;

    return 0;
}
/*
    output
    -------
    enter your firstname : Prakash
    enter your age : 27
    firstname : Prakash
    age : 27
    
*/